e2e/steps_file.js

```diff
@@ -194,6 +202,16 @@ module.exports = function () {
       await this.completeEvent('Save and continue');
     },
 
+    async fillDate(date, sectionId = 'form'){
```

Comment:
spacing on bracket again

Reply:
also does this need to be async?

Reply:
Pulled this in from @tomaszpowroznik branch, not sure on whether it needs to be or not